coralibre-sdk/sdk/src/androidTest/java/org/coralibre/android/sdk/internal/AESBenchmark.java

```diff
@@ -49,7 +49,7 @@ public AESBenchmark(int size) {
 	public BenchmarkRule benchmarkRule = new BenchmarkRule();
 
 	@Test
-	public void micro_create_ephIDs() throws InvalidKeyException, NoSuchAlgorithmException, NoSuchPaddingException {
```

Comment:
:D well, this part gets stripped later on. It's part of dp3t, so we don't need it anyway.